guvnor-project/guvnor-project-backend/src/main/java/org/guvnor/common/services/project/backend/server/ProjectResourcePaths.java

```diff
@@ -0,0 +1,21 @@
+package org.guvnor.common.services.project.backend.server;
```

Comment:
Looks like we really should add checkstyle/enforcer config which would fail the build in case of missing (or malformed) license headers. @mbiarnes could you take a look at that when you have some time?